ppoprf/benches/bench.rs

```diff
@@ -115,8 +114,8 @@ fn benchmark_server(c: &mut Criterion) {
 
 fn benchmark_client(c: &mut Criterion) {
   let mds: Vec<u8> = (0..=7).collect();
-  let input =
-    digest::digest(&digest::SHA512, &Scalar::random(&mut OsRng).to_bytes());
+  let mut input = [0u8; 32];
+  PRNG.fill_bytes(&mut input);
```

Comment:
One of the most valuable things you can do with your commit message is to document *why* a change was made. The 'what' is nice to summarise, but can generally be inferred from the patch itself. Recording motivation makes it much easier to reason about later changes to the same code, since the concerns and requirements that went into a change are much more elusive when just reading the code.

You do this for not zeroing the GCM key or client data, but not the ServerPublicKey or the change from ring's SHA512 (as described in the RFC?) to a strobe hash. I'd like to see the commit message updated with some explanation since this in particular seems like a major change.